index.html

```diff
@@ -2,394 +2,275 @@
             </li>
-          </ul> 
-                            
-        </section> 
-              
-        <section class=language-wrap>
-          <section class="get">
-          <button class= "fill"> Get my resume </button> 
-          </section> 
-
-          <section class="language">
-            <ul>
-              <li>Languages</li>
-              <li><img class="arrow-down" src="imgs/arrow-down.png" alt="" /></li>
-            </ul>
-          </section>
-
-          <section class="prog">
-            <section class="box prog-js">
-              <ul>
-                <li><img class="sty" src="imgs/ellipse-js.png" alt="javaScript" /></li>
-                <li class="prog-sty">JavaScript</li>
-              </ul>
-            </section>
-
-            <section class="box prog-html">
-              <ul>
-                <li>
-                  <img class="sty" src="imgs/ellipse-html.png" alt="javaScript"/>
-                </li>
-                <li class="prog-sty">HTML</li>
-              </ul>
-            </section>
-
-            <section class=" box prog-css">
-              <ul>
-                <li><img class="sty" src="imgs/ellipse-css.png" alt="css" /> </li>
-                <li class="prog-sty">CSS</li>  
-              </ul>
-            </section>
-          </section>
-          
-          <section class="frameworks">
-            <ul>
-              <li>Frameworks</li>
-              <li><img class="arrow-head" src="imgs/arrow-head.png" alt="" /></li>
-            </ul>
-          </section>
-
-        <section class="separator">
-          <img src="imgs/separator.png" alt=" Separator" />
-        </section>
-
-        <section class="skills">
-          <ul>
-            <li class="ski">Skills</li>
-            <li><img class="arrow-head" src="imgs/arrow-head.png" alt="" /></li>
           </ul>
-        </section>
-
-        <section class="separator">
-          <img src="imgs/separator.png" alt=" Separator" />
-        <section>                   
-      </section> 
-    </section>   
-    <section class="last"> 
-      <section class="contacts-form">
-
-        <section class="contact-me">
-          <h3>Contact me</h3>
-        </section>
-        
-        <section class="last-content">
-          <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-        </section class="last-form">
-
-        <form action="https://formspree.io/f/xqkwojbj" class="form-contact" method="post">
-          <ul>
-            <li><input type="text" placeholder="Yeremias" name="name" id="name_form" maxlength="30" required/></li>
-            <li> <input type="email" placeholder="natajayanj14@gmail.com" name="email" id="email_form" required/> </li>
-            <li> <textarea name="message" placeholder="Write your message here"id="message" cols="30" rows="10" maxlength="500" required ></textarea></li>
```

Comment:
- [ ] Every form fields and forms in the page should have basic validations, you can read more about it [here](https://www.w3.org/WAI/test-evaluate/preliminary/#forms).

Reply:
Field that are required have been clearly been indicated, and asterisks * included in the field label.